docs/developer-docs/latest/developer-resources/database-apis-reference/rest/relations.md

```diff
@@ -41,6 +41,12 @@ You can also use the longhand syntax to [reorder relations](#relations-reorderin
 
 `connect` can be used in combination with [`disconnect`](#disconnect).
 
+:::warning
```

Comment:
```suggestion
:::caution
```

Warning is the highest-level alert. It should only be used for cases that break the app or delete data :)